src/foundations/shape/Shape.types.ts

```diff
@@ -3,7 +3,9 @@ import { TContainerElement, TInteractionElement } from '../../shared/types';
 // TODO: maybe move the hard-coded ones to a separate union type (`TActionElement` maybe?)
 type TShapeElement = TContainerElement | TInteractionElement;
 
-type TShapeBorderRadius = 0 | 4 | 8 | 12 | 16 | 20 | 24 | 'circle' | 'pill';
+type TShapeVariant = 'rectangle' | 'circle' | 'pill';
+
+type TShapeBorderRadius = 0 | 2 | 4 | 8 | 12 | 16 | 20 | 24 | 'circle' | 'pill';
```

Comment:
if we have the variant prop we might not need the border radius to be 'circle' or 'pill' anymore, and keep the type as number only and just add `SHAPE_BORDER_RADIUS_MAPPING` or something similar. WDYT?

Reply:
I want to prevent adding more props than necessary. IMO this is sufficient and as @deanwhillier already mentioned: 'We should not care about the complexity underneath, just make the usage as easy as possible for the users.' ✨